README.md

```diff
@@ -17,7 +19,16 @@ Sockeye requires:
 - numpy
 
 ## Installation
-You have two options for installing sockeye: pip and directly from source.
+
+### For AWS DeepLearning AMI users
+
+AWS DeepLearning AMI users need to run only one line in order to install the sockeye:
```

Comment:
"AWS DeepLearning AMI users only need to run the following line to install sockeye"